docs/cni/kubernetes/vagrant-coreos/VagrantCoreOS.md

```diff
@@ -8,110 +8,9 @@
 
```

Comment:
This master only banner references a non-existent image.  We should probably just remove the "latest version of docs" banner(s, if there are any more).